addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -75,7 +75,7 @@ def _domain_holiday_status_id(self):
         'Duration (days)', compute='_compute_number_of_days_display',
         help="For an Accrual Allocation, this field contains the theorical amount of time given to the employee, due to a previous start date, on the first run of the plan. This can be manually edited.")
     number_of_hours_display = fields.Float(
-        'Duration (hours)', compute='_compute_number_of_hours_display',
+        'Duration (hours)', compute='_compute_number_of_hours_display', store=True, readonly=False,
```

Comment:
- Though unrelated to the task, do you know why the field is `readonly=False`? Makes no sense IMO
- I believed you modified the report to use a `number_of_hours` field that was added in your refactor, but it's not in this file? I'm kinda lost :sweat_smile:

Reply:
- I think this was to stay consistent with when this field wasn't stored, and non-readonly being the default for non-stored computed fields, but indeed it doesn't really make sense, I'll change it
- the report uses number_of_hours from hr.leave, but still uses number_of_hours_display for allocation, I didn't add a number_of_hours for hr.leave.allocation.